mirrord/cli/src/db_branches.rs

```diff
@@ -0,0 +1,176 @@
+use kube::{
+    Api, Client,
+    api::{DeleteParams, ListParams},
+};
+use mirrord_operator::crd::mysql_branching::MysqlBranchDatabase;
+use prettytable::{Table, row};
+
+use crate::{
+    CliResult,
+    config::{DbBranchesArgs, DbBranchesCommand},
+};
+
+pub async fn db_branches_command(args: DbBranchesArgs) -> CliResult<()> {
+    match &args.command {
+        DbBranchesCommand::Status { names } => status_command(&args, names.clone()).await,
+        DbBranchesCommand::Destroy { all, names } => {
+            destroy_command(&args, *all, names.clone()).await
+        }
+    }
+}
+
+async fn status_command(args: &DbBranchesArgs, names: Vec<String>) -> CliResult<()> {
+    let client = Client::try_default().await.map_err(|e| {
+        crate::CliError::CreateKubeApiFailed(mirrord_kube::error::KubeApiError::KubeError(e))
+    })?;
+
+    let api: Api<MysqlBranchDatabase> = if args.all_namespaces {
+        Api::all(client)
+    } else if let Some(namespace) = &args.namespace {
+        Api::namespaced(client, namespace)
+    } else {
+        Api::default_namespaced(client)
+    };
+
+    let list_params = ListParams::default();
+
+    let branches = api.list(&list_params).await.map_err(|e| {
+        crate::CliError::ListTargetsFailed(mirrord_kube::error::KubeApiError::KubeError(e))
+    })?;
+
+    // Filter by names if specified
+    let filtered_branches: Vec<_> = if names.is_empty() {
+        branches.into_iter().collect()
+    } else {
+        branches
+            .into_iter()
+            .filter(|branch| {
+                branch
+                    .metadata
+                    .name
+                    .as_ref()
+                    .is_some_and(|name| names.contains(name))
+            })
+            .collect()
+    };
+
+    if filtered_branches.is_empty() {
+        println!("No active DB branch found");
```

Comment:
I saw the operator status command does `println!` too, but if we add a `ProgressTracker`, this can be `progress.success(Some("No active DB branch found."));`